routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -85,13 +89,28 @@ func ListTrackedTimes(ctx *context.APIContext) {
 		IssueID:      issue.ID,
 	}
 
+	qUser := strings.Trim(ctx.Query("user"), " ")
+	if qUser != "" {
+		user, err := models.GetUserByName(qUser)
+		if err != nil {
+			ctx.Error(http.StatusInternalServerError, "GetUserByName", err)
+			return
+		}
+		opts.UserID = user.ID
```

Comment:
We should detect the permissions here, not only if it's an administrator? The users who have permissions to write issue should also could do that.

Reply:
I'm just copying the behaviour of `repoTrackedTimes` here.

I agree that this permission check is weird, because tracked times are publicly visible in the web UI. So I'm not sure why this admin-filter was there. And checking for manage-issues-permission is not really better here.

Reply:
I do not know why i had to add it ... there where some complains ...

anyway I'm fine if you would just remove that  check ...

Reply:
Thinking about it, I'm hesitant to remove this check, from a privacy perspective. Hmm 🤔
(It's one thing to have this info displayed in the gui, but to have it scrapable with a single request per repo across all gitea instances? idk man..)

Reply:
OK. That should be another PR's thing I think.